homeassistant/components/tado/water_heater.py

```diff
@@ -0,0 +1,303 @@
+"""
+Support for Tado hot water zones.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/water_heater/tado/
```

Comment:
We don't need the link in the docstring anymore. Just keep the very first line.

Reply:
:+1: